examples/htool_console.c

```diff
@@ -222,10 +231,14 @@ int htool_console_run(struct libhoth_usb_device *dev,
   if (opts->history) offset -= 0x80000000;
   bool quit = false;
   while (!quit) {
-    status = read_console(dev, opts, &offset);
+    size_t response_size;
+    status = read_console(dev, opts, &offset, &response_size);
     if (status != LIBHOTH_OK) {
       break;
     }
+    if (response_size == 0) {
+      usleep(10000);
```

Comment:
Can I propose something a bit different?

What do you think of having a new option to the console, --log, which does the following:

- stops listening to stdin
- polls once a second instead of once every 10ms
- pastes a timestamp after every newline character

I can implement this if you like...

Reply:
The other thing you might want it to do is to retry when dauntless restarts and re-enumerates over USB

Reply:
Thanks for the review. Yes, "--log" looks good and would make a lot of sense if it did not read from stdin. Thanks in advance if you can do that.
I do not need a timestamp after every newline character. I did implement that earlier (I still have the patch handy) but turns out I can also do that with simply piping through `ts +%s`.

Reply:
"retry when the RoT restarts" ... I can currently retry in the script so when `htool console` terminates I can just respawn it, because the USB address is anyway changing.